assignments/lambda-classes.js

```diff
@@ -1 +1,265 @@
 // CODE here for your Lambda Classes
+
+// ============================ CLASSES ===================================
```

Comment:
wow, amazing job with this one, I love the use of comments here. I hope classes made more sense to you. I loved it when we did this one. Awesome job at going really deep with the characters too.